job.go

```diff
@@ -16,15 +17,24 @@ func (j Jobs) Swap(i, n int)      { j[i], j[n] = j[n], j[i] }
 func (j Jobs) Less(i, n int) bool { return j[i].Name < j[n].Name }
 
 // return new array of jobs matching name filter
-func (j Jobs) Filter(f []string) Jobs {
+func (j Jobs) Select(f []string) (Jobs, error) {
 	var n Jobs
-	for i := 0; i < len(j); i++ {
-		if inArray(f, j[i].Name) {
-			n = append(n, j[i])
+
+	for _, name := range f {
+		found := false
```

Comment:
it might make more sense to label each for loop, and rather than using `found` your break call jumps to the outer labelled loop.

or possibly it might make sense to split the inner loop in to a private function.